### PR TITLE
BrickHack 8 Site - About Section

### LINKED ISSUE #1248: BrickHack 8 Site - About Section
Part of #1246 

Content can be found in Notion [here](https://www.notion.so/brickhackclub/BrickHack-8-Site-Content-9aa4b49ff06140068b293046546ff720)

Design can be found in Figma [here](https://www.figma.com/file/ZR9Mb4xNRrnoRZoX0bCWsX/BrickHack-Websites?node-id=432%3A97)

To do notes:
- No gallery for now
- [ ] cards
- [ ] top content
- [ ] content under cards (flexbox with right being image and left being content and another flex for those stats *GET STATS*)

### PR DESCRIPTION
**Pull request by sjmiller7**
Fixes #1248 

Had to remove prizes cards since we technically don't know if we have prizes yet.

At the time of posting, I am waiting on Design to fix the bar underneath the rickies. I might talk to them Monday (12/6) about switching to gifs for quality, since the video here is also fuzzy. Might need to do some analysis of gif file size vs quality upgrade.

### CODE REVIEW

**Comment by clairegallup** [comment]
no netlify :pensive:

File: sass/index.scss [hunk 1]
```diff
@@ -234,34 +234,40 @@ section {
...
+
+            #stats {
+                display: flex;
+                justify-content: space-between;
+                flex-wrap: wrap;
+
+                div {
+                    padding: 20px;
+                }
+            }
```
Comment (clairegallup) [inline sass/index.scss:266]: The Figma design shows the leftmost statistic as being left-aligned with the left side of the container, but this line breaks that.

Figma:
image

Current implementation:
image

Here's a possible fix that seems to work well, but you can test it for yourself and make changes as needed:
```suggestion
            #stats {
                display: flex;
                gap: 20px;
                justify-content: space-between;
                padding: 20px 20px 20px 0;
                flex-wrap: wrap;
            }
```

File: index.html [hunk 2]
```diff
@@ -64,25 +64,45 @@
...
+                <h3>Participating</h3>
+                <p>At BrickHack, students compete individually or in teams of 2 - 4 to build anything they want. You could make an app, a website, a game, a design, or any other form of interactive media. No experience is required; we provide mentors to help you along the way!</p>
+            </div>
+        </div>
+        <div id="about-stats">
+            <div id="stats-content">
+                <p>Last year's BrickHack 7 in numbers:</p>
+                <div id="stats">
+                    <div>
+                        <h3>600</h3>
```
Comment (clairegallup) [inline index.html:90]: content nit: Figma shows each statistic ending with an exclamation point:
image

Is that something we're going to add to the final content?

File: index.html [hunk 3]
```diff
@@ -64,25 +64,45 @@
...
-                    <i class="fas fa-arrow-right"></i>
-                    Visit Major League Hacking
-                </a>
-                <a href="./gallery.html" rel="noopener noreferrer">
-                    <i class="fas fa-images"></i>
-                    BrickHack Gallery
-                </a>
+            <div class="card">
+                <h3>Participating</h3>
+                <p>At BrickHack, students compete individually or in teams of 2 - 4 to build anything they want. You could make an app, a website, a game, a design, or any other form of interactive media. No experience is required; we provide mentors to help you along the way!</p>
```
Comment (clairegallup) [inline index.html:82]: content nit: Maybe get rid of the spaces between the numbers and hyphens for "2 - 4"? I usually see it without spaces.
```suggestion
                <p>At BrickHack, students compete individually or in teams of 2-4 to build anything they want. You could make an app, a website, a game, a design, or any other form of interactive media. No experience is required; we provide mentors to help you along the way!</p>
```

**Review by clairegallup — APPROVED** [review]
Looks good! I left a few nits about staying true to the Figma mockup, but nothing super important or necessary.

File: index.html (hunk 2 above)
Comment (sjmiller7) [reply index.html:90]: Need to ask Marketing really

File: index.html (hunk 3 above)
Comment (sjmiller7) [reply index.html:82]: Personal writing style choice. Fixing it

File: sass/index.scss (hunk 1 above)
Comment (sjmiller7) [reply sass/index.scss:267]: I fixed it a bit better in the next update

File: sass/index.scss [hunk 4]
```diff
@@ -512,6 +491,34 @@ footer {
...
+
+    // About
+    #about {
+        #about-cards {
+            flex-direction: column;
+
+            .card {
+                width: 100%;
+
+                &:not(.card:last-of-type) {
```
Comment (peterkos) [inline sass/index.scss:503]: Is this needed anymore?

**Review by peterkos — APPROVED** [review]
LGTM! A few nits: 

CleanShot 2021-12-19 at 00 04 04

**Comment by peterkos** [comment]
One thing we can tackle later is the extra padding in the right card due to less content compared to the left card. Can prob fix with some filler text

File: sass/index.scss (hunk 4 above)
Comment (sjmiller7) [reply sass/index.scss:503]: Removing it makes the two cards press up against each other with no space between them.

**Comment by sjmiller7** [comment]
> less padding here?
See design's Figma. That is specified.

> maybe shift this section up a bit?
See design's Figma. That is specified. Moving it up would push it to flex-start and create a ton of whitespace underneath.

> Numbers can be bigger
Last update I made to number styling was doing the code in front of design team and asking what they liked. They specifically approved what is in there now.

If there are issues to be had with what design team is specifying, I recommend pinging them about it in #design channel. I am inclined to go with whatever they say.

> Can prob fix with some filler text
Not really sure what you mean here. It would be something to bring up with marketing in the general pr. Generally, we should want our content to be concise for ux/skimming purposes, not verbose for aesthetic purposes.

**Comment by peterkos** [comment]
Apologies for being a bit out of the loop; as with the numbers changing from the figma, I wasn't 100% on if the spacing was also as intentional.  

>> Can prob fix with some filler text
> Not really sure what you mean here. It would be something to bring up with marketing in the general pr.

Yea, that's what I mean -- would be good to talk w/ Adam on adding some filler as an alternative to modifying the design itself

LGTM

**Comment by skyegallup** [comment]
I agree with @sjmiller7 that it's not a great idea to add unnecessary or unnecessarily verbose text just to eliminate whitespace. Content > styling

**Pull request merged**